src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -48,14 +51,34 @@ private static bool IsInvalidDouble(double number)
                 Func<IRContext, int, FormulaValue, FormulaValue> checkRuntimeTypes,
                 Func<IRContext, int, FormulaValue, FormulaValue> checkRuntimeValues,
                 ReturnBehavior returnBehavior,
-                Func<EvalVisitor, EvalVisitorContext, IRContext, T[], ValueTask<FormulaValue>> targetFunction)
+                Func<EvalVisitor, EvalVisitorContext, IRContext, T[], ValueTask<FormulaValue>> targetFunction,
+                bool isMultiArgTabularOverload = false)
             where T : FormulaValue
         {
             return async (runner, context, irContext, args) =>
             {
                 var nonFiniteArgError = FiniteArgumentCheck(functionName, irContext, args);
                 if (nonFiniteArgError != null)
                 {
+                    (var maxSize, var minsSize, var emptyTablePresent, _, var errorTable) = AnalyzeTableArguments(args, irContext);
```

Comment:
>emptyTablePresent [](http://example.com/codeflow?start=52&length=17)

I don't see this value being used anywhere; if that's the case, we should remove it from the result of AnalyzeTableArguments